.gitignore

```diff
@@ -164,3 +164,6 @@ docs/mzm_generation/data/
 
 # plot directories
 docs/mzm_generation/plots/
+
+# Binary file from running notebook
+docs/mzm_generation/.DS_Store
```

Comment:
It looks like `.DS_Store` is a MacOS-specific file that can appear anywhere, so this line should simply be

```
.DS_Store
```